plugins/inputs/snmp/snmp.go

```diff
@@ -149,6 +131,45 @@ func (s *Snmp) init() error {
 	return nil
 }
 
+func (s *Snmp) getMibsPath() {
+	gosmi.Init()
+	var folders []string
+	for _, mibPath := range s.Path {
+		gosmi.AppendPath(mibPath)
+		folders = append(folders, mibPath)
+		err := filepath.Walk(mibPath, func(path string, info os.FileInfo, err error) error {
+			if info.Mode()&os.ModeSymlink != 0 {
+				link, err := os.Readlink(path)
+				if err != nil {
+					s.Log.Warnf("Bad symbolic link %v")
+				}
+				folders = append(folders, link)
+			}
```

Comment:
It seems like here you are adding all symlinks to the folders. I wonder what happens if I put a symlink to a file in the `mibPath` folder. Anyway, can you please add a short comment on why you _only_ process symlinks here. Why is it needed?

Reply:
I added a comment!